lib/infomon/infomon.rb

```diff
@@ -94,6 +98,26 @@ def self.set(key, value)
     self.upsert(key: self._key(key), value: self._validate!(key, value))
   end
 
+  def self.blob_set(*blob)
```

Comment:
Why did we add another method that does the same thing? Also we should not be wrapping this in `Thread.new` like that because we have now created a race conditon